crates/diesel_models/src/relay.rs

```diff
@@ -0,0 +1,77 @@
+use common_utils::pii;
+use diesel::{AsChangeset, Identifiable, Insertable, Queryable, Selectable};
+use time::PrimitiveDateTime;
+
+use crate::{enums as storage_enums, schema::relay};
+
+#[derive(
+    Clone,
+    Debug,
+    Eq,
+    Identifiable,
+    Queryable,
+    Selectable,
+    PartialEq,
+    serde::Serialize,
+    serde::Deserialize,
+)]
+#[diesel(table_name = relay)]
+pub struct Relay {
+    pub id: common_utils::id_type::RelayId,
+    pub connector_resource_id: String,
+    pub connector_id: common_utils::id_type::MerchantConnectorAccountId,
+    pub profile_id: common_utils::id_type::ProfileId,
+    pub merchant_id: common_utils::id_type::MerchantId,
+    pub relay_type: storage_enums::RelayType,
+    pub request_data: Option<pii::SecretSerdeValue>,
```

Comment:
Do we have an option to store structured data (serialized as JSON) in the `request_data` and `response_data` fields?